common/app/views/fragments/image.scala.html

```diff
@@ -33,33 +35,57 @@
     }
 }
 
-<picture>
-    @* IE 9 needs this workaround as per https://scottjehl.github.io/picturefill/ *@
-    <!--[if IE 9]><video style="display: none;"><![endif]-->
+@* Nb. assumes lazy loading JS is active *@
+@if(shouldLazyLoad && Switches.LazyLoadImagesSwitch.isSwitchedOn) {
```

Comment:
Note, the if/else branches here involve a lot of repetition. I'm planning to try and share stuff here a bit but thoughts on this welcome.